vnext/PropertySheets/JSEngine.props

```diff
@@ -8,7 +8,7 @@
     <!-- Enabling this will (1) Include hermes glues in the Microsoft.ReactNative binaries AND (2) Make hermes the default engine -->
     <UseHermes Condition="'$(UseHermes)' == ''">false</UseHermes>
     <!-- This will be true if (1) the client want to use hermes by setting UseHermes to true OR (2) We are building for UWP where dynamic switching is enabled -->
-    <HermesVersion Condition="'$(HermesVersion)' == ''">0.70.0</HermesVersion>
+    <HermesVersion Condition="'$(HermesVersion)' == ''">0.70.1</HermesVersion>
```

Comment:
Don't forget to update the couple of packages.config files that have this hard-coded.